dts/arm/st/stm32f446.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2017 Pushpal Sidhu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/stm32f407.dtsi>
```

Comment:
This inclusion is not valid:
stm32f407 includes ethernet/HASH_RNG/FPU IPs which are not included in stm32f446.

Use the following instead:
#include <st/stm32f401.dtsi>

Then add USART3 and UART4 UART5 support in this file.
Plus you need to add matching -pinctrl file.